### ISSUE TITLE
RSS Feed

### ISSUE THREAD

**Issue by nlvw** [kzahel/jstorrent-legacy-app#69]
Is it possible to add an RSS feed option similar to what is in utorrent or qbittorrent?

**Comment by kzahel** [comment]
No, I haven't added this functionality. I don't really have any plans to anytime soon. I never really had a use for it personally. I was always annoyed that uTorrent only kept the last RSS response, instead of continually adding items to the RSS feed list (i.e. saving all the responses in a database)

Would you use it for auto-downloading ? What is the minimal feature that you need?

**Comment by nlvw** [comment]
The feature is mostly for auto downloading.  Such as when new anime/manga
episodes come out the client automatically grabs the new episode from the
feed and puts it in the designated folder.

If you look at qbittorrent's implementation its very robust and easily
organized.  They also have it in another "tab" so that its out of the way
unless you actually use it.

**Comment by kzahel** [comment]
Thanks for the tip. I'll take a look at their implementation and see how much work it would take. I thought that's what most people use the functionality for (auto-download).
I'm also not sure if it's yet possible for the app to run in the background even when the app is not running e.g. to check the RSS feed every few hours. I'll have to look into that too.

**Comment by kzahel** [comment]
No RSS support yet, but it's something I could add down the road. The new JSTorrent is rebuilt from scratch and ready at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues or have feature requests, please leave feedback!
